File: VietOCR/trunk/src/net/sourceforge/vietocr/GuiWithImage.java
package net.sourceforge.vietocr;

import javax.swing.JOptionPane;

public class GuiWithImage extends GuiWithCommand {

    @Override
    void readImageMetadata() {
        if (iioImageList == null) {
            JOptionPane.showMessageDialog(this, "Please load an image.");
            return;
        }
        
        ImageInfoDialog dialog = new ImageInfoDialog(this, true);
        dialog.setImage(iioImageList.get(imageIndex));
        dialog.setVisible(true);
    }
}

File: VietOCR.NET/trunk/GUIWithImage.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace VietOCR.NET
{
    public partial class GUIWithImage : VietOCR.NET.GUIWithCommand
    {
        public GUIWithImage()
        {
            InitializeComponent();
        }

        protected override void metadataToolStripMenuItem_Click(object sender, EventArgs e)
        {
            if (imageList == null)
            {
                MessageBox.Show("Please load an image.");
                return;
            }

            ImageInfoDialog dialog = new ImageInfoDialog();
            dialog.Image = imageList[imageIndex];

            if (dialog.ShowDialog() == DialogResult.OK)
            {

            }
        }

        protected override void screenshotModeToolStripMenuItem_Click(object sender, EventArgs e)
        {
            ToolStripMenuItem mi = (ToolStripMenuItem)sender;
            mi.Checked ^= true;
        }
    }
}
